src/components/Phone.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import style from 'styled-components';
+
+const PhoneComponent = style.p`
+  display: flex;
+  margin-top: -10px;
+  margin-left: 957px;
+  width: 51px;
+  left: 1282px;
+  top: 1320px;
+  font-family: 'Poppins';
+  font-style: normal;
+  font-weight: 500;
+  font-size: 16px;
+  line-height: 24px;
+  color: #333333;
+  width: 90px;
+  left: 1236px;
+  top: 1257px;
+  background-color: lightcoral;
+  border-radius: 40px;
+  padding: 3px 2px 2px 2px;`;
+const Phone = ({ type, className, onClick, children, id }) => {
+  <PhoneComponent>
+    type={type ? type : 'p'}
```

Comment:
Why is this `type` used here? Please remove it

Reply:
Deteled type={type ? type : 'p'}